src/lsdb/dask/divisions.py

```diff
@@ -0,0 +1,31 @@
+from typing import List, Tuple
+
+from hipscat.pixel_math import HealpixPixel
+from hipscat.pixel_math.hipscat_id import healpix_to_hipscat_id
+
+
+def get_pixels_divisions(ordered_pixels: List[HealpixPixel]) -> Tuple[int, ...]:
+    """Calculates the Dask Dataframe divisions for a list of HEALPix pixels.
+
+    Divisions include the minimum value of every HEALPix pixel hipscat_id
+    and the maximum value of the last HEALPix pixel hipscat_id. In practice
+    they represent the bounds of hipscat_id values for the target pixels.
+
+    Args:
+        ordered_pixels (List[HealpixPixel]): The list of HEALPix pixels to
+            calculate the hipscat_id bounds for. They must be ordered by
+            ascending hipscat_id.
+
+    Returns:
+        The Dask Dataframe divisions, as a tuple of integer.
+    """
+    divisions = []
+    for index, hp_pixel in enumerate(ordered_pixels):
```

Comment:
You're right!! I missed it because `healpix_to_hipscat_id` arguments are typed as `int`. I'm going to open a PR in `hipscat` to introduce the `HIPSCAT_ID_MAX` and I will make sure these types are also fixed. Also, what do you think of making `HIPSCAT_ID_MAX` 2**64-1, for simplicity?